spdx-utils/src/main/resources/exceptions/Autoconf-exception-3.0

```diff
@@ -2,7 +2,7 @@ AUTOCONF CONFIGURE SCRIPT EXCEPTION
 
 Version 3.0, 18 August 2009
 
-Copyright © 2009 Free Software Foundation, Inc. <http://fsf.org/>
+Copyright � 2009 Free Software Foundation, Inc. <http://fsf.org/>
```

Comment:
Introduces character encoding issue for (C)

Reply:
Thanks, turns out there indeed was an encoding issue.